guides/06_Meeting_Events.md

```diff
@@ -248,6 +252,22 @@ JavaScript uses the browser's [`getUserMedia` API](https://developer.mozilla.org
 
 <br>
 
+### Audio transmission related events
+
+The Javascript SDK publishes the following events to report issues with audio transmission:
+
+- The `receivingAudioDropped` event is triggered when the packet loss for receiving audio is high. This can be used to warn users about potential network connectivity issues but if you are already using the [connectionDidSuggestStopVideo](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html#connectiondidsuggeststopvideo) or [connectionDidBecomePoor](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideoobserver.html#connectiondidbecomepoor) callbacks you do not need to take any action as these callbacks are triggered on the same Predicate.
+
+- The `sendingAudioFailed` event is triggered when audio packets are not being sent out consistently. This event can be used to alert the user that their audio is not reaching other attendees or to ask them to take a manual action such as changing the input audio device or restarting the application. When there's a recovery in the audio packets sent, the `sendingAudioRecovered` event is published which can be used to acknowledge that the user action did succeed.  Unlike the `receivingAudioDropped` event there are no additional callbacks which are invoked for this case. There are few caveats attached to these events listed below:
+  - The `sendingAudioFailed` event may not trigger accurately in Firefox when the user switches from a working audio input device to a non-working one, this is due to a potential bug in the WebRTC's getStats() implementation within Firefox. 
```

Comment:
How should a builder compensate for this?

Reply:
Do you mean, how should builders handle the events or are you specifically asking about the Firefox issue?

Reply:
I think the prior if I understand that correctly.

For the firefox issue, should we then file a bug with them? Changing audio device failure affects the user experience in a meeting hence if this occurs would be good to track it as a browser bug or provide a workaround in such a case.

Reply:
Got it, I'm figuring out how to file an issue with FF with reproducible steps in a standalone manner

Reply:
After some more deep dive realized that this is not actually a bug with Firefox's `getStats()` implementation. Rather, it's just the way we're testing  that's causing the disparity between browsers. 

Chromium's implementation for an empty MediaStreamDestination doesn't generate any audio packets whereas in Firefox it always defaults to sending frames of silence. (I verified this on the server side by logging the packets as they're received)

I've just about tried every possible way to induce a "no audio" scenario using Web Audio APIs but had no success. Here are my findings:
- Any kind of `AudioScheduledSourceNode` such as [AudioBufferSourceNode](https://developer.mozilla.org/docs/Web/API/AudioBufferSourceNode), [ConstantSourceNode](https://developer.mozilla.org/docs/Web/API/ConstantSourceNode) and [OscillatorNode](https://developer.mozilla.org/docs/Web/API/OscillatorNode) do not work as the documentation clearly states they output silence when not playing. 
- Tried adding a DelayNode with a large delay on top of these to try and induce "no audio" temporarily but even when the delay is in effect silent audio packets are generated.
-  Also looked at AudioWorkletNode with custom AudioWorkletProcessor but the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/AudioWorkletProcessor) states that the output of the node by default holds silence. I tried deleting the values in the array but that throws an error too.

So, I'm just going to remove this line from the guide and call out that the` No Audio` device doesn't work in Firefox in the demo app

Reply:
Checking if we have considered not using Web Audio APIs and use MediaStream APIs? Or that also does not help?

Reply:
Yes, I have tried using MediaStream API first. But, I couldn't find a good way to create a dummy MediaStreamTrack that produces no audio. Even the hacky ways mentioned [here](https://github.com/web-platform-tests/wpt/issues/7424) like `new RTCPeerConnection().addTransceiver('audio').receiver.track` don't seem to do the trick because FF still sends silent packets. There are many ways to mock a MediaStreamTrack but so far none of them worked while running the application for me

Also, we can't get away with not passing a MediaStreamTrack by simple constructing an empty MediaStream, because we have an explicit [condition to throw an error](https://github.com/aws/amazon-chime-sdk-js/blob/main/src/audiovideocontroller/DefaultAudioVideoController.ts#L1056-L1058) when that happens